MessagePrompt.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+
+namespace CMP1124M_Assignment
+{
+    public class MessagePrompt
+    {
+        public string Message { get; private set; }
+        public MessagePrompt(string message)
+        {
+            Message = message;
+        }
+
+        public void ShowMessage()
+        {
+            Console.WriteLine(Message);
+        }
+
+        public int ShowMessageNumericalResponse(int min, int max)
```

Comment:
Should add the ability to add options (index + name) which are shown automatically.